### PR TITLE
Added a space to avoid additional args breaking the app port.

### PR DESCRIPTION
**Pull request by pedrotski**
# Description

Adding an additional argument would break the app port as it would be added like this

Example: 

+app.port 28083-logfile

 +app.port 28083 -logfile

adding the space fixes that.

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [X] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]
Dont just edit the json file. Edit it in the panel and export it.

**Comment by lilkingjr1** [comment]
> Dont just edit the json file. Edit it in the panel and export it.

May I ask why? They just added a single, needed, space character; properly too. Looks fine to me (assuming it's the only thing that needs changing).

**Comment by QuintenQVD0** [comment]
> > Dont just edit the json file. Edit it in the panel and export it.
> 
> May I ask why? They just added a single, needed, space character; properly too. Looks fine to me (assuming it's the only thing that needs changing).

Then the egg goos to pltv_2 format (if your panel is up to date) and then match the export date the date of this PR

**Comment by lilkingjr1** [comment]
> > > Dont just edit the json file. Edit it in the panel and export it.
> > 
> > 
> > May I ask why? They just added a single, needed, space character; properly too. Looks fine to me (assuming it's the only thing that needs changing).
> 
> Then the egg goos to pltv_2 format (if your panel is up to date) and then match the export date the date of this PR

Ahh, didn't notice the egg was still PTDL_v1. Makes sense.

**Comment by pedrotski** [comment]
Totally replaced it to the new version. Turns out the stock eggs here are outdated.

**Comment by Software-Noob** [comment]
The stock eggs are not to be up to date with the upstream as mentioned in the readme. That's not what they are meant for.

Honestly they should be removed all together due to constant confusion and only leave readme with a link to upstream. Any PRs to upstream can be done through branches.

**Comment by QuintenQVD0** [comment]
> The stock eggs are not to be up to date with the upstream as mentioned in the readme. That's not what they are meant for.
> 
> Honestly they should be removed all together due to constant confusion and only leave readme with a link to upstream. Any PRs to upstream can be done through branches.

I will have a look later this day to just remove the stock eggs here and 1 readme that points to the eggs in the panel repo

**Comment by QuintenQVD0** [comment]
if this https://github.com/parkervcp/eggs/pull/1945 get merged then this pr is not more needed. But keep it open for now

File: stock_eggs/rust/egg-rust.json [hunk 1]
```diff
@@ -147,16 +157,38 @@
...
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|url",
+            "field_type": "text"
+        },
+        {
+            "name": "Custom Map URL",
+            "description": "Overwrites the map with the one from the direct download URL. Invalid URLs will cause the server to crash.",
+            "env_variable": "MAP_URL",
```
Comment (parkervcp) [inline stock_eggs/rust/egg-rust.json:186]: You added more than just a space.

**Review by parkervcp — CHANGES_REQUESTED** [review]

**Comment by parkervcp** [comment]
Actually if this is a stock egg make the PR there.

**Pull request closed without merging**